Nos/Models/NosNotification+CoreDataClass.swift

```diff
@@ -76,4 +70,15 @@ public class NosNotification: NSManagedObject {
             try? context.saveIfNeeded()
         }
     }
+
+    static func oldNotificationsRequest() -> NSFetchRequest<NosNotification> {
+        let fetchRequest = NSFetchRequest<NosNotification>(entityName: "NosNotification")
+        let since = cutoffDate()
+        fetchRequest.predicate = NSPredicate(format: "createdAt == nil OR createdAt < %@", since as CVarArg)
+        return fetchRequest
+    }
+
+    static func cutoffDate() -> Date {
```

Comment:
This is named very similarly to `PushNotificationService.notificationCutoff` but performs a slightly different function which I think I will find confusing in the future.
- `NosNotification.cutoffDate()` is two months before `Date.now`, and is used to delete old notifications from the db.
- `PushNotificationService.notificationCutoff` is used to limit which notifications are displayed to the user as push notifications. When the user first opens the app it is initialized to Date.now. This is to prevent us showing tons of push notifications on first login. Then after that it is set to the date of the last notification that we showed. 

Could we make these names more specific and maybe document the two vars? I think this one could be `staleNotificationCutoff` and the other could be `showPushNotificationsAfter`, but I'm open to other ideas too.